wayland/install-wayland.txt

```diff
@@ -0,0 +1,158 @@
+may vary, but they typically do the heavy lifting from grabbing input events to
+allocating system resources and arrange all that for the user in a usable GUI.
+
+
+Caveats
+________________________________________________________________________________
+
+Aside from the fact that Wayland and it's implementations are in an early state
+of evolvement and Xorg still defines the GUI standard regarding stability,
+compatibility and portability, the biggest downside is choice!
+The different Wayland implementations are followed by lock ins towards supported
+hardware, device management and software for certain use cases which might not
+be portable across other implementations. Not all software needs may be
+satisfied yet.
+
+NOTE: This list is rather incomplete and might change, but it gives an idea on
+      what to expect.
+
+- Limited hardware and driver support. [8] [9]
+- Fragmentation accross the different Wayland implementations lead to special, 
+  non portable software for bar support, screen sharing, screen recording, ...
+  [10] [11] [12]
+- To extend compatibility it is possible to start applications with XWayland, an
+  underlying Xserver, but this brings the advertisement "aiming as a simple re-
+  placement for X" ad absurdum. [13]
+- Not every graphical application _just works_.
+- SDL has no Wayland backend. SDL 2 games with non free assets tend to not work.
+- One might need to set SUID on the compositor or elogind for user
+  authentication. [14]
+- Depending on the Wayland implementation, there is no free choice in the device
+  manager nor the absence of it at all. [15]
+- ...
+
+
+Why would I use it then?
+________________________________________________________________________________
+
+That is entirely up to you. Aside from just trying out something new, a Wayland
+based GUI _can_ simplify your system. Screen tearing is not possible by design
+and hardware accelerated rendering may happen in places which are not covered by
+Xorg.
+
+- less moving parts [13], although many graphical programs still heavily rely on
+  Xorg libraries. (Mesa [16], Firefox [17], Qt-WebEngine [18], ...) 
+- No screen tearing.
+- Hardware accelerated video playback in Firefox. (VAAPI)
+- Hardware accelerated rendering in webkit2gtk. (WPEBackend-fdo)
+- If you can delimit your needs and expectations on KISS in the aforementioned 
+  caveats.
+
+
+Prerequisites
+________________________________________________________________________________
+
+Though KISS does not support Wayland officially there is no problem in making it
+capable through personal or 3rd party repositories. [19] The official
+repositories satisfy all dependencies. After &lt;wayland&gt; and &lt;wayland-protocols&gt;
+are installed, some package adjustments are required to make them to known
+citizens of your system.
+
+NOTE: From this point it is expected that &lt;wayland&gt; and &lt;wayland-protocols&gt; 
```

Comment:
I think it's better to do `"wayland"` and `"wayland-protocols"` instead of using `&lt;`/`&gt;`.

Reply:
You could drop the quotes/<> entirely too.

Reply:
I think the reference to an actual package can be misleading when only expressed through capitalization.